### PR TITLE
MentionBadge FF

### PR DESCRIPTION
**Pull request by Velin92**
Requires a new SDK release with the notificationCount fix

### CODE REVIEW

File: ElementX/Sources/Services/Room/RoomSummary/RoomSummaryProvider.swift [hunk 1]
```diff
@@ -238,14 +238,14 @@ class RoomSummaryProvider: RoomSummaryProviderProtocol {
         }
         
         let notificationMode = roomInfo.userDefinedNotificationMode.flatMap { RoomNotificationModeProxy.from(roomNotificationMode: $0) }
-        
+        MXLog.info("roomInfo: \(roomInfo)")
```
Comment (stefanceriu) [inline ElementX/Sources/Services/Room/RoomSummary/RoomSummaryProvider.swift:241]: Let's get rid of this please, too much info, especially for very large accounts

**Review by stefanceriu — APPROVED** [review]
lgtm but it's still a question whether it's the right way to go

**Comment by Velin92** [comment]
> lgtm but it's still a question whether it's the right way to go

let's wait, for me and Benoit this doesn't work on this SDK version sadly so maybe not...

**Review by stefanceriu — APPROVED** [review]
lgtm 👍

**Pull request merged**